kinbot/reaction_generator.py

```diff
@@ -337,11 +337,6 @@ def generate(self):
                                         prod_opt = obj.prod_opt[j]
                                         break
 
-                    elog=open("energy.log",'a')
-                    for prod_opt in obj.prod_opt:
-                        elog.write("prod_opt: {} |\tenergy: {}\n".format(prod_opt.species.chemid, prod_opt.species.energy))
-                    elog.close()
-
```

Comment:
Why is this deleted?

Reply:
I believe that was for testing purposes to see if the right energies were being recorded. I can add it back in, but I'm not sure that the output file is necessary for people to have.